srcts/src/shiny/notifications.ts

```diff
@@ -67,9 +80,8 @@ function show({
   return id;
 }
 
-// TODO-barret - Should `id` be required? (some places do not supply one)
-function remove(id?: string): void {
-  get(id).fadeOut(fadeDuration, function () {
+function remove(id: string): void {
+  get(id)?.fadeOut(fadeDuration, function () {
```

Comment:
Should `shinyUnbindAll()` be called if there is no matching object for `get(id)`?